File: crates/aegis-language/tests/corpora/python/negatives.py
# negatives.py — negative / narrowness corpus: references without calls, and
# comments or strings that merely mention a tracked API.
#
# Expected: zero operations. A comment or string literal that mentions
# `os.remove` is not a call; an attribute reference without a call (`os.remove`
# bound to a name) is not an operation; unrelated calls (print, len) are not
# tracked destructive or execution-sink APIs. Parses cleanly.
import os

# A comment mentioning the API is not a call.
# os.remove("/tmp/x")

# A string mentioning the API is not a call.
doc = "call os.remove('/tmp/x') to delete"

# An attribute reference without a call is not an operation.
handler = os.remove

# Unrelated calls are not tracked operations.
print(doc)
len(doc)

File: crates/aegis-language/tests/corpora/javascript/negatives.js
// Corpus: negatives — constructs that must NOT surface as operations (plan
// Iteration 7, ADR-022 §3).
//
// Pins the adapter's narrowness: a destructive API spelling inside a comment
// or string literal, a member reference with no call, an unrelated call, and a
// CommonJS/ESM import (imports are resolution input only and emit no operation
// themselves — bounded symbol resolution is a later slice). Each must parse
// cleanly and yield zero operations.

// fs.unlinkSync("x");
"fs.unlinkSync('x')";
const ref = fs.unlinkSync;
console.log("hello");
import fs from "fs";